### ISSUE TITLE
Missing shop data for some food and drink items

### ISSUE THREAD

**Issue by alistair3149** [StarCitizenWiki/API#76]
[Black Mountain Sujin Tea (Medium Blend)](https://api.star-citizen.wiki/api/v2/items/e040cb26-8ea0-44ae-bac4-00bee0f55dd5?include=shops.items) and [Breakfast dog](https://api.star-citizen.wiki/api/v2/items/1af97566-2ca7-4ac8-ad4f-f7ec6421f331?include=shops.items) is sold in game but there are no shop data in the API. Could be more items but those are the two that I found.

**Comment by octfx** [comment]
This is currently a limitation of the underlying Unpacked data. :/

All food and drink items are sold in shops of which the position can only be found in `Layer0.xml`

I've started to manually add them here: https://github.com/StarCitizenWiki/API/blob/dbd9234f28f4064e165edcd3cdb58fc27a11636f/app/Services/Parser/SC/Shops/Shops.php#L180  
So the easiest way would be to add the "most important" shops there.

While I did add all Layer0 information to [starmap.json](https://github.com/StarCitizenWiki/scunpacked-data/blob/master/starmap.json), I hadn't had the time to combine the information from `starmap.json` and `shops.json`.
